public/editor/scripts/project/project.js

```diff
@@ -237,22 +238,48 @@ define(function(require) {
             return callback(err);
           }
 
-          // Find an HTML file to open in the project, hopefully /index.html
-          var sh = new _fs.Shell();
-          sh.find(getRoot(), {name: "*.html"}, function(err, found) {
+          // Find the index.html file in the project root to open
+          _fs.readdir(getRoot(), function(err, found) {
```

Comment:
I would rename `found` to `children`, since `readdir()` will return all children of a dir vs. some query (like `find` does).